src/main/java/by/itech/lab/supplier/exception/ConflictWithTheCurrentStateException.java

```diff
@@ -0,0 +1,11 @@
+package by.itech.lab.supplier.exception;
+
+public class ConflictWithTheCurrentStateException extends RuntimeException{
+
+
+    private static final long serialVersionUID = 3743903100951638522L;
+
+    public ConflictWithTheCurrentStateException(String message) {
```

Comment:
Can we use more informative name for a class? To clarify what state is. "..CurrentWarehouseState.." for example.